pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -534,6 +768,16 @@ func (ds *SQLPlugin) openDB(cfg *configuration) (*gorm.DB, error) {
 	return db, nil
 }
 
+// addCommonLabels append common labels for this entity to the input labels,
+// and return the result
+func (ds *SQLPlugin) addCommonLabels(labels ...telemetry.Label) []telemetry.Label {
+	return append(labels, telemetry.Label{Name: telemetry.DatabaseType, Value: ds.db.databaseType})
+}
+
+func (ds *SQLPlugin) prepareMetricsForCall(ctx context.Context, labels ...telemetry.Label) telemetry.Metrics {
```

Comment:
this is always called with the results of `addCommonLabels` as the argument.... i think we should probably just fold addCommonLabels into this method so the callsites aren't so verbose.